CPT331.Core/Logging/OutputStreams.cs

```diff
@@ -0,0 +1,31 @@
+﻿#region Using References
+
+using System;
+using System.Diagnostics;
+
+#endregion
+
+namespace CPT331.Core.Logging
+{
+	public static class OutputStreams
```

Comment:
New type for logging information to various output streams. Might be able to include some logging to the file system here too.